ComfyUI/comfy_api_nodes/apis/client.py

```
@@ -346,31 +356,53 @@ def request(
+        final_headers = {**default_headers, **headers}
+
+        print(f"[DEBUG] Request URL: {url}")
+        print(f"[DEBUG] Method: {method}")
+        print(f"[DEBUG] Headers before request: {final_headers}")
+        print(f"[DEBUG] Content-Type: {content_type}")
 
         # Special handling for Ideogram API which requires multipart/form-data
         if "api.ideogram.ai" in self.base_url:
             content_type = "multipart/form-data"
-            print(f"[DEBUG] Ideogram request data: {data}")
+            print(f"[DEBUG] Processing Ideogram request")
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)